examples/mithril.ts

```diff
@@ -3,7 +3,7 @@ export interface Vnode {
   tag: String|Object;
   key?: String;
   attrs?: Object;
-  children?: Array|String|Number|Boolean;
+  children?: Array<any>|String|Number|Boolean;
```

Comment:
this a short term workaround? `Array` should imply `Array<any>`

Reply:
when it was running the compiler this was causing a TS error so i changed it to this, but maybe in newer TS versions this is valid?

<img width="394" alt="image" src="https://github.com/progrium/reflect-ts/assets/5448363/4a0d6fa0-7508-4ece-bf08-05dda1df9d2a">